cypress/component/TermsOfService/tos_acceptance.spec.js

```diff
@@ -1,18 +1,36 @@
 /* eslint-disable no-undef */
 
 import React from 'react';
-import { mount } from 'cypress/react';
+import {mount} from 'cypress/react';
+import {Auth} from '../../../src/libs/auth/auth';
 import TermsOfServiceAcceptance from '../../../src/pages/TermsOfServiceAcceptance';
-import { ToS } from '../../../src/libs/ajax/ToS';
-import {Storage} from '../../../src/libs/storage';
 import {Navigation} from '../../../src/libs/utils';
+import {OAuth2} from '../../../src/libs/ajax/OAuth2';
+import {ToS} from '../../../src/libs/ajax/ToS';
+import {Storage} from '../../../src/libs/storage';
 
 const text = 'TOS Text';
 const mocks = {
-  history: { push() {} }
+  history: {
```

Comment:
Nit: Is this a lint change? I personally find it more readable the other way.

Reply:
Yes, that's it exactly - I manually unformatted it, then ran our eslint checks and made the same change.